### PR TITLE
Add server_info_ip to create server.info with specific ip addr

### PR DESCRIPTION
**Pull request by Yingshun**
Some cases need to create a special server.info with specific ip
address.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
`1/1) type_specific.io-github-autotest-libvirt.virt_admin.server_update_tls: PASS (119.75 s)`

File: virttest/utils_conn.py [hunk 1]
```diff
@@ -902,7 +902,7 @@ class TLSConnection(ConnectionBase):
                  'credential_dict', 'qemu_tls', 'qemu_chardev_tls',
                  'server_saslconf', 'server_qemuconf', 'client_qemuconf',
                  'server_libvirtd_tls_socket', 'client_libvirtd_tls_socket',
-                 'special_cn', 'server_setup_local',
+                 'special_cn', 'server_setup_local', 'server_info_ip',
```
Comment (chunfuwen) [inline virttest/utils_conn.py:905]: you may need add description after line 893 for your newly added attribute

Comment (Yingshun) [reply virttest/utils_conn.py:906]: Yes, updated.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**